Add sms schedules tutorial (#274)
medic/cht-docs: content/en/apps/tutorials/sms-schedules.md

---
title: "Building SMS Forms"
linkTitle: SMS Schedules
weight: 6
description: >
  Building CHT application SMS forms
relatedContent: >

>   apps/reference/app-settings/schedules

Reviewer: a couple additional related content resources:
apps/concepts/forms
apps/features/messaging
apps/guides/messaging

Author: Added these as well.

---

> SMS schedules allow you to send reminder messages at predetermined times. SMS schedules can be triggered by [SMS forms]({{< ref "apps/tutorials/sms-forms" >}}) or App forms.

Reviewer: It would be helpful to tell the reader why this feature can be useful.

Suggest moving this to the `key concepts` section
> SMS schedules can be triggered by [SMS forms]({{< ref "apps/tutorials/sms-forms" >}}) or App forms.

Author: Done.

## Purpose of the Tutorial

> This tutorial will take you through how to set up SMS schedules for CHT applications.

Reviewer: ```suggestion
This tutorial takes you through how to set up SMS schedules for CHT applications. It uses a pregnancy registration workflow and follow-up reminders for a Community Health Worker as an example. The same methodology can be applied to other workflows and reminders as needed. 
```

> You will be setting up pregnancy follow up reminders for a pregnancy registration workflow that allows Community Health Workers to register new pregnancies for the household members and get reminders to follow up on the pregnancy.

Reviewer: I pulled this sentence into the previous line, so it can be deleted if you like the change

Author: Done. Yeah, the updated one is more succinct.

            "content": "Hello {{contact.name}}, please remind {{patient_name}} ({{patient_id}}) to go for her clinic visit this week.",
            "locale": "en"
          }
        ],
        "group": 2,
        "offset": "51 days",
        "send_day": "",
        "send_time": "09:00",
        "recipient": "reporting_unit"
      },
      {
        "message": [
          {
            "content": "Hello {{contact.name}}, please remind {{patient_name}} ({{patient_id}}) to go for her clinic visit this week.",
            "locale": "en"
          }
        ],
        "group": 3,
        "offset": "79 days",
        "send_day": "",
        "send_time": "09:00",
        "recipient": "reporting_unit"
      },
      {
        "message": [
          {
            "content": "Hello {{contact.name}}, please remind {{patient_name}} ({{patient_id}}) to go for her clinic visit this week.",
            "locale": "en"
          }
        ],
        "group": 4,
        "offset": "107 days",
        "send_day": "",
        "send_time": "09:00",
        "recipient": "reporting_unit"
      }
    ]
  }
]
```

{{< see-also page="apps/reference/app-settings/schedules" title="Schedule properties" >}}

> ### 2. Assign the schedule

Reviewer: Capitalized to follow format

```suggestion
### 2. Assign the Schedule
```

---
title: "Building SMS Forms"
linkTitle: SMS Schedules
weight: 6
description: >
  Building CHT application SMS forms
relatedContent: >
  apps/reference/app-settings/schedules
  apps/concepts/forms
  apps/features/messaging
  apps/guides/messaging

---

> SMS schedules allow you to send reminder messages at predetermined times. These reminders serve as useful prompts for end-users to take specific actions. For example, a community health worker receiving an SMS reminding them to follow up on a pregnant woman who is due for her monthly clinic visit.

Reviewer: I thought the last sentence was covered in the "purpose" section so removed it
```suggestion
SMS schedules allow you to send reminder messages at predetermined times. These reminders serve as useful prompts for end-users to take specific actions.
```

## Purpose of the Tutorial

This tutorial takes you through how to set up SMS schedules for CHT applications. It uses a pregnancy registration workflow and follow-up reminders for a Community Health Worker as an example. The same methodology can be applied to other workflows and reminders as needed.

## Brief Overview of Key Concepts

*[SMS schedules]({{< ref "apps/reference/app-settings/schedules" >}})* are a series of SMS messages that are to be sent to specific contacts at future dates and times.

> SMS schedules can be triggered by *[SMS forms]({{< ref "apps/tutorials/sms-forms" >}})* or *App forms*.

Reviewer: ```suggestion
SMS schedules can be triggered by *[SMS forms]({{< ref "apps/tutorials/sms-forms" >}})* or *[App forms]({{< ref "apps/reference/forms/app" >}})*.
```
